wgengine/magicsock/magicsock.go

```diff
@@ -3310,6 +3372,7 @@ type endpoint struct {
 	c          *Conn
 	publicKey  key.NodePublic // peer public key (for WireGuard + DERP)
 	fakeWGAddr netip.AddrPort // the UDP address we tell wireguard-go we're using
+	nodeAddr   netip.Addr     // the tailscale node address (only used for logging purposes)
```

Comment:
// the node's first Tailscale address (only used for logging)

Reply:
Done.